p2panda-store/src/lib.rs

```diff
@@ -1,5 +1,19 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
+//! Interfaces and implementations of persistence layers for core p2panda data types.
+//!
+//! The provided APIs allow for efficient implementations of `Operation` and `Log` stores. These
```

Comment:
```suggestion
//! The provided APIs allow for efficient implementations of `Operation` and log stores. These
```

Don't think we have a code-equivalent called `Log`.